### DISCUSSION TITLE
Paper writeup

### DISCUSSION THREAD

**Discussion by seabbs** [epiforecasts/EpiNow2#1099] [Q&A]
Maybe we should write up the main models, the workflow and the synthetic validation and stick it in a paper? It would certainly help with citations getting lost, it might help inspire work on the documentation, and it would help people feel safe.

We could make it seem like science by doing some evaluation on the synthetic recovery with different settings (probably couldn't cover the other models as that would make it rather long).

Before doing that I would want to rewrite `estimate_delays()` and potentially `estimate_truncation()` (to use a slightly more valid but still simple nowcasting model).

**Comment by jamesmbaazam** [comment]
Had a discussion with @sbfnk about some potential project ideas stemming out of #695 (currently outdated vs local copy to be pushed soon). I would suggest reviving this conversation and outlining what could go in the paper and commence work on it.

**Comment by jamesmbaazam** [comment]
During August, I'll pool together the documentation of the EpiNow2 models and other docs in preparation for a draft EpiNow2 paper. Let's set up a meeting in September when Seb is back, to discuss: the outline, venue, contents, etc.
